md-commons/src/main/java/com/wipro/ats/bdre/md/app/StoreJson.java

```diff
@@ -0,0 +1,40 @@
+package com.wipro.ats.bdre.md.app;
+
+import java.util.List;
+
+/**
+ * Created by cloudera on 3/10/16.
+ */
+public class StoreJson {
+    private String name;
+    private String id;
+
+    public List<AppValues> getAppValuesList() {
+        return appValuesList;
+    }
+
+    public void setAppValuesList(List<AppValues> appValuesList) {
+        this.appValuesList = appValuesList;
+    }
+
+    private List<AppValues> appValuesList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)